python/ctsm/site_and_regional/plumber_site.py

```diff
@@ -0,0 +1,124 @@
+"""
+This module contains the Plumber2Site class and class functions which are used in run_tower.py
+"""
+
+# Import libraries
+import logging
+import os
+import sys
+
+# Get the ctsm util tools and then the cime tools.
+_CTSM_PYTHON = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "python"))
+sys.path.insert(1, _CTSM_PYTHON)
+
+# -- import local classes for this script
+# pylint: disable=wrong-import-position
+from ctsm.site_and_regional.tower_site import TowerSite
+
+# pylint: disable=wrong-import-position, import-error, unused-import, wrong-import-order
+from ctsm import add_cime_to_path
+from ctsm.path_utils import path_to_ctsm_root
+
+from CIME import build
+from CIME.case import Case
+from CIME.utils import safe_copy, expect, symlink_force
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=too-many-instance-attributes
+class Plumber2Site(TowerSite):
+    """
+    A class for encapsulating plumber sites.
+    """
+
+    def build_base_case(
+        self,
+        cesmroot,
+        output_root,
+        res,
+        compset,
+        user_mods_dirs=None,
+        overwrite=False,
+        setup_only=False,
+    ):
+        if user_mods_dirs is None:
+            user_mods_dirs = [
+                os.path.join(self.cesmroot, "cime_config", "usermods_dirs", "PLUMBER2", self.name)
+            ]
+        case_path = super().build_base_case(cesmroot, output_root, res, compset, user_mods_dirs)
+
+        return case_path
+
+    # pylint: disable=too-many-statements
+    def run_case(
+        self,
+        base_case_root,
+        run_type,
+        prism,
```

Comment:
we won't support prism for PLUMBER2 sites, but it's needed for the run_tower command, with `./run_tower --help` noting this is only supported for NEON sites.